plenum/test/common/test_replicas_suspicious.py

```diff
@@ -0,0 +1,34 @@
+from plenum.common.constants import DOMAIN_LEDGER_ID
+
+from plenum.test.helper import waitForViewChange
+from stp_core.loop.eventually import eventually
+
+
+def test_nodes_make_view_change_only_on_master_suspicious(
+        looper, txnPoolNodeSet):
+    old_view = txnPoolNodeSet[0].viewNo
+
+    primary1 = txnPoolNodeSet[0].replicas[0]
```

Comment:
Better variable names will be `primary_master` and `primary_backup`

Reply:
ok